app/views/candidate_interface/start_page/not_eligible.html.erb

```diff
@@ -10,6 +10,6 @@
     <p class="govuk-body"><%= service_name %> is still in development, so for now, we have to limit candidate numbers.</p>
     <p class="govuk-body govuk-!-font-weight-bold">You can apply for all teacher training courses and providers using UCAS.</p>
     <p class="govuk-body">You’ll need to register with UCAS before you can apply.</p>
-    <p class="govuk-body"><%= govuk_link_to 'Apply on UCAS', UCAS.apply_url %></p>
+    <p class="govuk-body"><%= govuk_link_to t('apply_from_find.ucas_apply_button'), UCAS.apply_url, class: 'govuk-button' %></p>
```

Comment:
You can use `govuk_button_link_to` instead here.

Reply:
Ah, lovely. Updated.